sourmash/sourmash_args.py

```diff
@@ -402,6 +402,21 @@ def _load_database(filename, traverse_yield_all, *, cache_size=None):
     except Exception as exc:
         pass
 
+    if not loaded:   # try load signatures from single file (list of signature paths)
+        try:
+            db = []
+            with open(filename, 'rt') as fp:
+                for line in fp:
+                    line = line.strip()
+                    if line:
+                        sig = sourmash.load_signatures(line, quiet=True, do_raise=True)
```

Comment:
I implemented it in `_load_database`, but the file list could potentially contains indices too (not only paths to signatures). Should this be a test case? (it wouldn't be too hard to change, I could change this line to use `load_file_as_signatures` instead...)

Reply:
Implemented in 7e95ad24c349d602ae86b7584337b79cc8cad5e5